cmd/promtool/main.go

```diff
@@ -56,6 +56,13 @@ import (
 	"github.com/prometheus/prometheus/scrape"
 )
 
+const (
+	successExitCode = 0
+	failureExitCode = 1
+	// exit code 3 is used for "one or more lint issues detected".
```

Comment:
nit: capitalize comment

```suggestion
	// Exit code 3 is used for "one or more lint issues detected".
```